### ISSUE TITLE
Matrix Singing Faces Not Consistently Installing on Target Prop During Import

### ISSUE THREAD

**Issue by MCStoff** [xLightsSequencer/xLights#4983]
**Describe the bug**
When importing a matrix singing face from a zipped zip file, the import algorithm is not installing the matrix singing face on the target prop.  The issue is inconsistent.  The algorithm works most of the time, but not all of the time.

**To Reproduce**
1. File/Import
2. Browse to target packaged xLights sequence.
3. Map source prop with matrix singing face to target prop.

**Expected behavior**
For a long time, xLights has consistently and properly installed the matrix singing face on the target prop.  The intermittent problem is just now coming to light as more people are importing sequences.

**Versions (please complete the following information):**
Because it is a problem reported to Showstopper Sequences, I cannot isolate at which version of xLights it stopped functioning as it previously had.   As soon as information becomes available, I will report the versions of xLights being used.

**Additional context**
I can provide a source sequence to any developer looking into the reported problem.  Customers have reported the problem with Christmas Vacation and That Time of Year.  Christmas Vacation was imported on Windows 11 using version 2023.03.  That Time of Year was imported on a Mac.  I will provide xLights version as soon as the user is back at his computer and can relay that information to me.

**Comment by cybercop23** [comment]
Hi Michael. I think we tried to replicate this several times with no avail. If you can still dupolicate it either share the full steps here and/or join us in the Zoom  room.

**Comment by MCStoff** [comment]
I am not the one experiencing the problem; I just reported it. My customers on both Windows and Mac OS have experience it on different versions of xLights. I cannot replicate it on my system either. But I have seen it occur first-hand in Zoom. I may be able to find someone who has experienced the problem who is willing to help replicate it if you are interested in that. Sent from my iPhoneOn Jan 8, 2025, at 12:58 PM, AlexB ***@***.***> wrote:﻿
Hi Michael. I think we tried to replicate this several times with no avail. If you can still dupolicate it either share the full steps here and/or join us in the Zoom  room.

**Comment by cybercop23** [comment]
Yes. We need a test sequence and someone that has the issue since we've tried to replicate it wih our own setups and it imports as expected.

**Comment by MCStoff** [comment]
I just sent out a query to one of the people that had the problem. He seemed pretty savvy on the computer. I will let you know as soon as he lets me know if he has time and is willing to help. I will query a  couple of the others who experienced the problem as well. Thank you. Sent from my iPhoneOn Jan 8, 2025, at 1:59 PM, AlexB ***@***.***> wrote:﻿
Yes. We need a test sequence and someone that has the issue since we've tried to replicate it wih our own setups and it imports as expected.

**Comment by MCStoff** [comment]
Ralph Braun has experienced the problem with not only Showstopper sequences but another vendor as well.  He sent me this as his contact information:  ***@***.*** Ph # 1(941)258-5580. He will be out of the country Jan 21 - 27, but is happy to help.
Thank you.

MichaelShowstopper Sequences
 

    On Wednesday, January 8, 2025 at 02:04:03 PM CST, Michael Stoffregen ***@***.***> wrote:  
 
 I just sent out a query to one of the people that had the problem. He seemed pretty savvy on the computer. I will let you know as soon as he lets me know if he has time and is willing to help. 
I will query a  couple of the others who experienced the problem as well. 

Thank you. 

Sent from my iPhone

On Jan 8, 2025, at 1:59 PM, AlexB ***@***.***> wrote:



﻿


Yes. We need a test sequence and someone that has the issue since we've tried to replicate it wih our own setups and it imports as expected.

**Comment by derwin12** [comment]
I did try with Polar Express and couldn't duplicate it .. must be something subtle that we are missing in trying to duplicate this.
(Is there another ticket in for this 'cus I feel like I have already mentioned this)

**Comment by MCStoff** [comment]
AlexB has been trying to duplicate it as well and couldn't.  I can't duplicate it on my computers.  Ralph has had the issue with more than just Showstopper sequences so he is a good candidate for replicating the problem in real time.  Intermittent problems are the worst.  Also, I think sequencers' and developers' computers tend to have faster processors with more RAM than the average user, and that may be a factor in why we can't duplicate the issue.

MichaelShowstopper Sequences
 

    On Sunday, January 12, 2025 at 08:25:12 AM CST, Daryl ***@***.***> wrote:  
 
 


I did try with Polar Express and couldn't duplicate it .. must be something subtle that we are missing in trying to duplicate this.
(Is there another ticket in for this 'cus I feel like I have already mentioned this)

**Comment by cybercop23** [comment]
unable to duplicate and no further feedback. Please reopen when more info is available.
